src/App.css

```diff
@@ -5,6 +5,26 @@
   text-align: center;
 }
 
+.navbar {
+  align-items: center;
+  flex-direction: column;
+  justify-content: left;
+  border: solid;
+  border-radius: 10px;
+  padding: 1rem;
+  margin-bottom: 4px;
+  background-color: bisque;
+
+}
+
+.navbar-item {
+  display:inline;
+  margin: 1rem;
+  padding: 4px;
+  font-weight: 550;
+  color: rgb(68, 12, 12);
+}
+
```

Comment:
I think a bit of time spent making the styling better would pay dividends! 🎨 
The navbar overlaps the content a bit, and the image is not responsive when using smaller screena
<img width="1484" alt="image" src="https://github.com/madDwain/nc-news/assets/151947206/e71c33f7-18b9-484b-aaf6-934220903aaf">
<img width="567" alt="image" src="https://github.com/madDwain/nc-news/assets/151947206/e5078a4a-af40-452f-8c36-9a0d06e6e51c">